src/main/java/com/ericsson/ei/jmespath/JmesPathInterface.java

```diff
@@ -72,8 +73,12 @@ public JsonNode runRuleOnEvent(String rule, String event) {
 
         try {
             Expression<JsonNode> expression = jmespath.compile(rule);
+            LOGGER.debug("EXPRESSION : " + expression );
```

Comment:
I'm not sure how much of this logging you want to keep, but if you want to ensure they are logged together, make one entry instead of several - since the many threads in Eiffel Intelligence makes it impossible to follow otherwise.

Reply:
removed debug loggers